src/api/logger.js

```diff
@@ -0,0 +1,57 @@
+import winston from "winston";
+
+// Define the severity levels
+const levels = {
+  error: 0,
+  warn: 1,
+  info: 2,
+  http: 3,
+  debug: 4,
+};
+
+/*
+ If the server was on development mode, show all the log levels.
+ If the server was on production mode, show only warn and error messages.
+*/
+const level = () => {
+  const env = process.env.NODE_ENV || 'development'
+  const isDevelopment = env === 'development'
+  return isDevelopment ? 'debug' : 'warn'
+};
```

Comment:
We can use .env variables to change the log levels in development as well as in production

Reply:
I see. I do not see it clearly on how to do so. Can you please show me?

Reply:
`const levels = {
  error: 0,
  warn: 1,
  info: 2,
  http: 3,
  debug: 4,
};`

`const Logger = winston.createLogger({
  level: process.env.LOG_LEVEL,
  levels,
  format,
  transports,
});`

Here we can change the value of level dynamically from env file. If env file has variable `LOG_LEVEL="debug`", This will log the `debug` and all the level above `debug`,

Reply:
```
const Logger = winston.createLogger({
  level: level(),
  levels,
  format,
  transports,
});
```

Here we the above as

```
const Logger = winston.createLogger({
  level: `process.env.LOG_LEVEL,
  levels,
  format,
  transports,
});
```